lib/lotus/helpers/html_helper.rb

```diff
@@ -184,8 +193,13 @@ module HtmlHelper
       #
       # @see Lotus::Helpers::HtmlHelper
       # @see Lotus::Helpers::HtmlHelper::HtmlBuilder
-      def html
-        HtmlBuilder.new
+      # @see Lotus::Helpers::HtmlHelper::HtmlFragment
+      def html(&blk)
+        if block_given?
+          HtmlBuilder.new.fragment(&blk)
```

Comment:
@jodosha thanks, do you think if it's viable to make it work this way?

```ruby
# html_builder.rb
def initialize(&blk)
  @nodes = []
  if block_given?
    @nodes = instance_eval(&blk).nodes
  end
end

attr_reader :nodes
```

Also `FormBuilder` raises conflict when we allow block as a parameter in `HtmlBuilder#initialize`. It could be bypassed by removing the `super()` invocation, but I can foresee one of disadvantages is about the reusability of `HTMLBuilder`.

```ruby
class FormBuilder
  def initialize(form, attributes, context = nil, &blk)
    @nodes = []
    # instead of super() invocation
    # ...
  end
end
```